### PR TITLE
Foundation JavaScript Fundementals-1: Fix/Add container to assignment section

### PR DESCRIPTION
**Pull request by Mclilzee**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

Add the proper container to Assignment section in foundations/javascript_basics/fundemantals-1.md
To make it more inline and constant with the assignments in the rest of the curriculum.

**3. Additional Information:**

### CODE REVIEW

File: foundations/javascript_basics/fundamentals-1.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Numbers are the building blocks of programming logic!  In fact, it's hard to thi
 
 ### Assignment
```
Comment (01zulfi) [inline foundations/javascript_basics/fundamentals-1.md:71]: The heading "Practice" feels more aligned with the content in this section as opposed to "Assignment". Can you change this as well?

**Review by 01zulfi — CHANGES_REQUESTED** [review]
Looks good. Just one change, then we're good to merge :rocket:

File: foundations/javascript_basics/fundamentals-1.md (hunk 1 above)
Comment (Mclilzee) [reply foundations/javascript_basics/fundamentals-1.md:71]: I had that in mind, However in the other fundementals the "Practice" has no container background like the Assignment sections. Would you have me change all the Practices to have a background or to change THIS practice and remove it's background to be more alligned with the rest?

Comment (01zulfi) [reply foundations/javascript_basics/fundamentals-1.md:71]: Thanks for asking. Practice sections did not use to have a div container. But with the new [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md), Practice should have one.

We're aware that some lessons have inconsistency in Practice sections as well as others. A curriculum wide project will deal with them soon. So for now, you don't need to change Practices in the other lessons. Just changing the heading from "Assignment" to "Practice" for this lesson should be enough.

Comment (Mclilzee) [reply foundations/javascript_basics/fundamentals-1.md:71]: I tracked down the 4 other practices and added background to them. If this going to interrupt an ingoing change, and you want me to revert them back and only change “Assignment” to “Practice” I will do so. However, if what you meant is, that I don't have to, but I can if I wanted to, then I will push the changes. Sorry for the inconvenience waiting an answer.

Comment (01zulfi) [reply foundations/javascript_basics/fundamentals-1.md:71]: Yes, please revert the other changes and only push the requested change for this PR. 

And it's not an inconvenience, I appreciate you asking! :smile:

**Review by 01zulfi — APPROVED** [review]
Thanks for contributing :fire:

**Pull request merged**